cli/state.rs

```diff
@@ -155,6 +155,9 @@ impl Loader for ThreadSafeState {
     referrer: &str,
     is_root: bool,
   ) -> Result<ModuleSpecifier, ErrBox> {
+    // Why are import maps not applicable to the root module?
```

Comment:
This is probably wrong... When I was implementing import maps I believe spec said that it shouldn't be applied to root module and main module of workers, but I can't find this bit now. I'll ask about it and get back to you.

Reply:
I changed it. Now they are still not applied to the main module but they are applied to dynamic imports

Reply:
Reference: https://github.com/WICG/import-maps/issues/157 - it seems that should be changed a bit, but that's a story for separate PR